api/v1alpha1/istio_types.go

```diff
@@ -44,6 +44,12 @@ type IstioSpec struct {
 	// +kubebuilder:validation:Schemaless
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Helm Values"
 	Values json.RawMessage `json:"values,omitempty"`
+
+	// RawValues defines the values to be passed to the Helm chart when installing Istio and that do not need a CRD validation.
+	// +kubebuilder:pruning:PreserveUnknownFields
+	// +kubebuilder:validation:Schemaless
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Helm RawValues"
+	RawValues json.RawMessage `json:"rawValues,omitempty"`
```

Comment:
Should we call this `unvalidatedValues` (that's what upstream calls it). Maybe it's better than `rawValues` since it may not be clear what "raw" means in this case?

Reply:
(but I don't really like "unvalidatedValues", because it's too long)

Maybe there's a better alternative than unvalidated or raw.

Reply:
Yeah. I'm thinkink maybe `rawValues` will make sense when we replace the `values` with actual types. On the other hand, outside of the operator, `values` already look like they're backed by actual types, so `rawValues` already makes sense.  

Maybe keep `rawValues` for now and then rename it later.